delocate/delocating.py

```diff
@@ -348,14 +348,12 @@ def _copy_required(
 
 def _dylibs_only(filename):
     # type: (Text) -> bool
-    return (filename.endswith('.so') or
-            filename.endswith('.dylib'))
+    return (filename.endswith('.so') or filename.endswith('.dylib'))
```

Comment:
I was surprised by this needed change; which rule is this?

I think we don't need the outer parentheses any more.

Reply:
The `W503,W504` codes disallow line-breaks before/after operators.  I fixed this section before deciding to ignore the rule.